tests/unit/htdocs/modules/system/SystemMaintenanceTest.php

```
@@ -261,4 +261,333 @@ public function isValidTableCachesResults(): void
+        $db->method('isResultSet')->willReturn(true);
+        $rows[] = false; // end-of-result sentinel
+        $db->method('fetchArray')->willReturnOnConsecutiveCalls(...$rows);
+    }
+
+    #[Test]
+    public function cleanAvatarRemovesValidAvatarFileUnderUploadRoot(): void
+    {
+        $scratchRel       = $this->avatarScratchRel();
+        [$rel, $abs]      = $this->placeFixtureAvatar($scratchRel, 'foo.png');
+
+        $db = $this->createMockDatabase();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)